### PR TITLE
More Raw Alcohol Recipes

### PR DESCRIPTION
**Pull request by Koollan**
Adds: Whiskey, Cognac, Vermouth, Gin, Tequila, Hessia (new alcohol), new Beer recipe and modifies other drink recipes. Adds Molasses as a reagent.

Also adds 2 jukebox .ogg files called Bluespace Ocean and This is Rain.

### CODE REVIEW

File: code/modules/reagents/recipes/sodas.dm [hunk 1]
```diff
@@ -24,26 +24,26 @@
...
 	catalysts = list(/datum/reagent/drink/juice/orange = 1) //You wanted orange but it gave you lemon-lime C:
 	mix_message = "The lime juice and lemon juice readily mix with the space cola while the orange juice forms a deposit. You feel disapointment."
 
 /datum/chemical_reaction/space_up
 	result = /datum/reagent/drink/space_up
 	result_amount = 3
-	required_reagents = list(/datum/reagent/drink/space_cola = 2, /datum/reagent/frostoil = 1) //helps keep your cool
+	required_reagents = list(/datum/reagent/drink/sodawater = 2, /datum/reagent/frostoil = 1) //helps keep your cool
 	catalysts = list(/datum/reagent/fuel = 1) //Tastes like a hull breach apparently so *shrug*
 	mix_message = "The frostoil and space cola mix together to form a thick liquid, as the fuel settles.. You thought you heard the air breach alarm just now.."
```
Comment (ingles98) [inline code/modules/reagents/recipes/sodas.dm:48]: change the 'space cola' accordingly i guess

**Comment by PsyCommando** [comment]
this is going to break several recipes. and it adds duplicates too.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -1435,6 +1435,15 @@
 	result_amount = 2
 	mix_message = "The solution thickens into a creamy white oil."
 
+/datum/chemical_reaction/molasses
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:1438]: this conflicts with honey crystalization.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 3]
```diff
@@ -1600,6 +1609,33 @@
 	result_amount = 10
 	mix_message = "The solution roils as it rapidly ferments into a rich red liquid."
 
+/datum/chemical_reaction/cognac
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:1612]: dont put new recipes randomly in this file. add them at the end or in the new recipe files. so your things dont get deleted by upstream merges

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 4]
```diff
@@ -269,6 +269,15 @@
 	nutriment_factor = 20
 	color = "#302000"
 
+/datum/reagent/nutriment/molasses
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:272]: move that to the bottom of the file please. Dont randomly slap it in the middle of everything.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 5]
```diff
@@ -1600,6 +1609,33 @@
...
+	required_reagents = list(/datum/reagent/ethanol/wine = 1)
+	minimum_temperature = 50 CELSIUS
+	maximum_temperature = (50 CELSIUS) + 100
+	result_amount = 1
+	mix_message = "The wine rapidly cooks off some of its properties and becomes clearer."
+
+datum/chemical_reaction/vermouth
+	name = "Vermouth"
+	result = /datum/reagent/ethanol/vermouth
+	required_reagents = list(/datum/reagent/ethanol/wine = 1, /datum/reagent/ethanol/vodka = 1)
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:1624]: pretty sure this is gonna break some drink mixes.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 6]
```diff
@@ -1632,13 +1668,22 @@
 	result_amount = 10
 	mix_message = "The solution roils as it rapidly ferments into a foaming amber liquid."
 
-/datum/chemical_reaction/spacebeer
-	name = "Space Beer"
-	result = /datum/reagent/ethanol/beer
+/datum/chemical_reaction/beer
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:1671]: there is already a beer recipe. and this conflict with the ale recipe.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 7]
```diff
@@ -1632,13 +1668,22 @@
...
+	name = "Beer"
+	result = /datum/reagent/ethanol/beer/good
+	required_reagents = list(/datum/reagent/nutriment/flour = 1, /datum/reagent/water = 1)
+	minimum_temperature = 50 CELSIUS
+	maximum_temperature = (50 CELSIUS) + 100
+	result_amount = 2
+	mix_message = "The solution roils as it rapidly ferments into a foaming amber liquid."
+
+/datum/chemical_reaction/tequilla
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:1680]: That recipe needs more reagents or itll break things made from corn oil

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 8]
```diff
@@ -2117,11 +2162,50 @@
...
 	name = "Rum"
 	result = /datum/reagent/ethanol/rum
-	required_reagents = list(/datum/reagent/sugar = 1, /datum/reagent/water = 1)
+	required_reagents = list(/datum/reagent/sugar = 1, /datum/reagent/water = 1, /datum/reagent/nutriment = 1)
 	catalysts = list(/datum/reagent/enzyme = 5)
-	result_amount = 2
+	result_amount = 3
 	mix_message = "The solution roils as it rapidly ferments into a red-brown liquid."
 
+/datum/chemical_reaction/whiskey
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:2170]: whiskey isnt made from molass only. Also there was already a whiskey recipe.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 9]
```diff
@@ -2117,11 +2162,50 @@
...
+	name = "Whiskey"
+	result = /datum/reagent/ethanol/whiskey
+	required_reagents = list(/datum/reagent/nutriment/molasses = 10)
+	catalysts = list(/datum/reagent/enzyme = 5)
+	result_amount = 10
+	mix_message = "The solution diminishes as it rapidly ferments into a clear brown liquid."
+
+/datum/chemical_reaction/starrycola
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:2178]: duplicate

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 10]
```diff
@@ -2117,11 +2162,50 @@
...
+	name = "Starry Cola"
+	result = /datum/reagent/ethanol/starrycola
+	required_reagents = list(/datum/reagent/ethanol/moonshine = 1, /datum/reagent/drink/space_cola = 2)
+	result_amount = 3
+
+datum/chemical_reaction/bluebird
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:2184]: duplicate

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 11]
```diff
@@ -2117,11 +2162,50 @@
...
+	name = "Blue Bird"
+	result = /datum/reagent/ethanol/bluebird
+	required_reagents = list(/datum/reagent/ethanol/gin = 1, /datum/reagent/ethanol/bluecuracao = 1, /datum/reagent/drink/juice/lemon = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/bj
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:2190]: duplicate

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 12]
```diff
@@ -2117,11 +2162,50 @@
...
+	name = "BJ"
+	result = /datum/reagent/ethanol/bj
+	required_reagents = list(/datum/reagent/ethanol/coffee/kahlua = 1, /datum/reagent/drink/milk = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/calvincraig
+	name = "Calvin Craig"
```
Comment (PsyCommando) [inline code/modules/reagents/Chemistry-Recipes.dm:2197]: duplicate

File: code/modules/reagents/recipes/alcohol.dm [hunk 13]
```diff
@@ -1,6 +1,7 @@
 /*
 	Base Alcohols
 */
+/* COMMENTED UNTIL NEW BREWING SYSTEM ADDED
```
Comment (PsyCommando) [inline code/modules/reagents/recipes/alcohol.dm:4]: dont comment out recipe while waiting on something.

File: code/modules/reagents/recipes/alcohol.dm [hunk 14]
```diff
@@ -10,36 +11,37 @@
 	maximum_temperature = T0C + 150
 	result_amount = 15
 	mix_message = "The solution roils as it rapidly ferments into a red-brown liquid."
-
+*/
 /*
 	Drink Mixes
-*/
+// SAME AS ABOVE
```
Comment (PsyCommando) [inline code/modules/reagents/recipes/alcohol.dm:17]: there is no reasons to comment out the recipes in this file only to splatter them into the master recipe file.

File: code/modules/reagents/recipes/sodas.dm [hunk 15]
```diff
@@ -24,26 +24,26 @@
 /datum/chemical_reaction/space_cola
 	result = /datum/reagent/drink/space_cola
 	result_amount = 5
-	required_reagents = list(/datum/reagent/drink/sodawater = 2, /datum/reagent/sugar = 4, /datum/reagent/glycerol = 1)
-	mix_message = "The clear soda water mixes with the sugar and thick glycerol to form a slightly thicker and darker liquid."
+	required_reagents = list(/datum/reagent/drink/sodawater = 2, /datum/reagent/sugar = 4, /datum/reagent/nutriment/molasses = 1)
```
Comment (PsyCommando) [inline code/modules/reagents/recipes/sodas.dm:27]: you're making a basic recipe too complicated and changed it for no good reasons

File: code/modules/reagents/recipes/sodas.dm [hunk 16]
```diff
@@ -24,26 +24,26 @@
...
-	required_reagents = list(/datum/reagent/drink/space_cola = 2,  /datum/reagent/drink/juice/lime = 2, /datum/reagent/drink/juice/lemon = 2)
+	required_reagents = list(/datum/reagent/drink/sodawater = 2,  /datum/reagent/drink/juice/lime = 2, /datum/reagent/drink/juice/lemon = 2)
 	catalysts = list(/datum/reagent/drink/juice/orange = 1) //You wanted orange but it gave you lemon-lime C:
 	mix_message = "The lime juice and lemon juice readily mix with the space cola while the orange juice forms a deposit. You feel disapointment."
 
 /datum/chemical_reaction/space_up
 	result = /datum/reagent/drink/space_up
 	result_amount = 3
-	required_reagents = list(/datum/reagent/drink/space_cola = 2, /datum/reagent/frostoil = 1) //helps keep your cool
+	required_reagents = list(/datum/reagent/drink/sodawater = 2, /datum/reagent/frostoil = 1) //helps keep your cool
```
Comment (PsyCommando) [inline code/modules/reagents/recipes/sodas.dm:46]: theres a reason why this was made from spacecola. to keep coherency with the rest of the colas recipes without creating conflicts

**Review by PsyCommando — CHANGES_REQUESTED** [review]
this pr is not gonna help us get rid of chem recipe conflicts and will add sevetal new ones. it also changes recipes we've had for a while for no good reason. honestly you should go back to the drawing board.

**Comment by PsyCommando** [comment]
you might want to make draft prs in the futuŕe if you're not sure about something

File: code/modules/reagents/recipes/sodas.dm (hunk 15 above)
Comment (Koollan) [reply code/modules/reagents/recipes/sodas.dm:27]: molasses is a lot easier to make than glycerol, and glycerol doesn't make much sense (it's a laxative, not a sweetener)

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 6 above)
Comment (Koollan) [reply code/modules/reagents/Chemistry-Recipes.dm:1671]: both recipes still work respectively in my testing

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm (hunk 4 above)
Comment (Koollan) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:272]: it's up near the top, where the nutriment class is.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 3 above)
Comment (Koollan) [reply code/modules/reagents/Chemistry-Recipes.dm:1612]: they are grouped together with the other hard alcohols.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 2 above)
Comment (Koollan) [reply code/modules/reagents/Chemistry-Recipes.dm:1438]: is that a new addition entirely? i was entirely unaware this existed

File: code/modules/reagents/recipes/sodas.dm (hunk 16 above)
Comment (Koollan) [reply code/modules/reagents/recipes/sodas.dm:46]: there was no comment saying that, so i figured it was a mistake that all of the not-cola drinks were made from cola so i changed them all to soda to be more accurate.

**Comment by Koollan** [comment]
apologies, i was just pretty sure about it all since in all of my testing, everything worked as intended and all of the 'conflicting' recipes also seemed to work so i'm just unsure of how to fix

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 7 above)
Comment (Koollan) [reply code/modules/reagents/Chemistry-Recipes.dm:1680]: this was the old beer recipe, unchanged, so this already was broken.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 2 above)
Comment (PsyCommando) [reply code/modules/reagents/Chemistry-Recipes.dm:1438]: That's something rhicora added

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 3 above)
Comment (PsyCommando) [reply code/modules/reagents/Chemistry-Recipes.dm:1612]: Yeah, but it's making everyone's life harder to have those there instead of the other file.

**Pull request closed without merging**